qa/workunits/rbd/qos.sh

```diff
@@ -21,14 +21,32 @@ rbd_bench() {
     fi
 
     # parse `rbd bench` output for string like this:
-    # elapsed:    25  ops:     2560  ops/sec:   100.08  bytes/sec: 409928.13
+    # elapsed:    25  ops:     2560  ops/sec:   100.08  bytes/sec: 409.13 MiB
     iops_bps=$(${timeout_cmd} rbd bench "${image}" \
                               --io-type ${type} --io-size 4K \
                               --io-total ${total} --rbd-cache=false \
                               --rbd_qos_${qos_type}_limit ${qos_limit} |
-                   awk '/elapsed:/ {print int($6) ":" int($8)}')
+                   awk '/elapsed:/ {print int($6) ":" int($8) " " $9}')
```

Comment:
nit (you can just ignore it): instead of processing it on shell side, you could just do all computations in awk, e.g. something like this:
```
awk '/elapsed:.*GiB/ {print int($6) ":" int($8) * 1024 * 1024 * 1024}
     /elapsed:.*MiB/ {print int($6) ":" int($8) * 1024 * 1024}
     /elapsed:.*KiB/ {print int($6) ":" int($8) * 1024}
     /elapsed:/      {print int($6) ":" int($8)}'
```

Reply:
I like your solution better -- tweaked